distributed/scheduler.py

```diff
@@ -6898,17 +6898,12 @@ def adaptive_target(self, comm=None, target_duration=None):
         if parent._unrunnable and not parent._workers_dv:
             cpu = max(1, cpu)
 
-        # Memory
-        limit_bytes = {
-            addr: ws._memory_limit for addr, ws in parent._workers_dv.items()
-        }
-        worker_bytes = [ws._nbytes for ws in parent._workers_dv.values()]
-        limit = sum(limit_bytes.values())
-        total = sum(worker_bytes)
-        if total > 0.6 * limit:
+        # add more workers if more than 60% of memory is used
+        limit = sum(ws._memory_limit for ws in parent._workers_dv.values())
```

Comment:
Some tests fail with:
`AttributeError: 'distributed.scheduler.WorkerState' object has no attribute '_memory_limit'`
The existing code should have the same issue?!

Reply:
mh, tests pass on the main branch even though it also accesses the same attribute? Any ideas @fjetter

Reply:
The failing test with python 3.7 + cython. cython doesn't seem to like `sum(generator)`. similar to the previous code, I'm now using `sum(list comprehension)`